src/components/Home.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable max-len */
+// import design from '../design/Home.css';
```

Comment:
- [ ] To keep your repo clean & professional , kindly remove any commented code from your code 🙏